packages/genesys-spark-components/src/components/beta/gux-time-zone-picker/gux-time-zone-picker.tsx

```diff
@@ -20,14 +20,20 @@ export class GuxTimeZonePickerBeta {
   @Element()
   private root: HTMLElement;
 
+  @Prop({ mutable: true })
+  value: string;
+
+  @Prop()
+  workspaceDefault: string;
+
   @Prop()
-  public value: string;
+  localDefault: string;
 
   @Prop()
-  public workspaceDefault: string;
+  customDefault: string;
 
   @Prop()
-  public localDefault: string;
+  customDefaultLabel: string;
```

Comment:
What's the difference between the `customDefaultLabel` prop and `customDefault`?

Reply:
The main difference between the two is that for example if you were to choose `localDefault` or `workspaceDefault` only one property is needed as the the string is hardcoded to either `Local Default` or `Workspace Default` and you just pass your time zone to the property whereas with the  two custom properties we would need 2 because I do not know what label the consumer will provide so we would need two properties one for the label and one for the timezone which they want to use. It would be nice to use property somehow but I dont think this is possible.

Reply:
Gotcha, thanks!